extensions/sql-migration/src/wizard/wizardController.ts

```diff
@@ -31,7 +31,7 @@ export class WizardController {
 	}
 
 	private async createWizard(stateModel: MigrationStateModel): Promise<void> {
-		const wizard = azdata.window.createWizard(loc.WIZARD_TITLE, 'wide');
```

Comment:
the second parameter is the dialog name used for telemetry not the width of the wizard.